src/components/ProgressIndicator/ProgressIndicatorDisplay.js

```diff
@@ -108,17 +110,37 @@ const ProgressIndicatorDisplay = ({
 }) => {
   const [expanded, setExpanded] = useState(false);
 
+  const intl = useIntl();
+  // Wrapper may be a DOM element, which can't handle <FormattedMessage />
+  const ariaIconLabel = intl.formatMessage({
+    description: 'Toggle active step progress indicator icon',
+    defaultMessage: 'Progress step indicator icon',
+  });
+  const toggleStepsTitle = intl.formatMessage({
+    description: 'Active step toggle titles',
+    defaultMessage: 'Show/hide steps',
+  });
+
   return (
     <Card blockClassName="progress-indicator" modifiers={expanded ? [] : ['mobile-collapsed']}>
-      <div
+      {/* Turn a Div with an onClick event into a button for accesibility */}
```

Comment:
@sergei-maertens I am assuming that: if a visually impaired user Zooms the screen, the mobile view will become triggered, where they should still be able to navigate the form-steps, so I am proposing to turn the div into a real button so it can have focus and can be operated with the space-bar etc.

Reply:
I checked, had to scroll to zoomlevel 200% before mobile view was triggered, but that is indeed what happens. Proposal is okay.